### ISSUE TITLE
Run several optimizations in parallel

### ISSUE THREAD

**Issue by jmcastro2109** [JuliaOpt/NLopt.jl#180]
Hi, I am writing here because I could not find this anywhere on the documentation (or in google FWIW)

Assume that I have to maximize N independent functions which are the "same" but there are some constant parameters (from the perspective of the optimization) that vary across each of the n \in N subproblems.

What would be the proper way of running each independent optimization in parallel? In other words, I would be calling NLopt in parallel to solve each subproblem n. What is the correct way of doing this? Since parameters are changing would I need to create n optimization routines?

**Comment by tbeason** [comment]
Since you are doing it in parellel, yes just create `N` different problems and submit them (presumably on different threads). From the optimizers point of view, the problems are not connected in any way, and since you are doing it simultaneously I don't think you can share any memory or anything like that.

**Comment by odow** [comment]
Closing as a duplicate of #146
